website/docs/r/dev_test_lab_schedule.html.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_dev_test_schedule"
+sidebar_current: "docs-azurerm-dev-test-lab-schedule"
+description: |-
+    Manages automated startup and shutdown schedules for Azure Dev Test Lab.
+---
+
+# azurerm_dev_test_schedule
+
+Manages automated startup and shutdown schedules for Azure Dev Test Lab.
+
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
```

Comment:
COuld we use `sample` instead of `test` for all resource names here
```suggestion
resource "azurerm_resource_group" "example" {
```

Reply:
Done